content/understand-django/2020-01-22-urls-lead-way.md

```diff
@@ -877,3 +869,17 @@ on Twitter
 where I am
 {{< extlink "https://twitter.com/mblayman" "@mblayman" >}}.
 {{< /web >}}
+{{< book >}}
```

Comment:
I can add this back. I didn't think it would useful since the next chapter immediately followed. I'd like to do this a little differently in the markup, but I'll keep the spirit of the change.